example/spec/helpers/testUtils.js

```diff
@@ -197,7 +197,17 @@ async function redeploy(config, options = {}) {
 }
 
 async function getFileMetadata(file) {
-  const { Bucket, Key } = parseS3Uri(file.filename);
+  let Bucket;
+  let Key;
+  if (file.bucket && file.filepath) {
+    Bucket = file.bucket;
+    Key = file.filepath;
+  }
+  else {
```

Comment:
https://github.com/nasa/cumulus/pull/800/files#diff-59617f11d03d238d58e5a4255941ea08R215
Why does this method still return `filename: file.filename` if `file.filename` may not be set.
Should we remove `else` block?

Reply:
The `getFileMetadata` function is used for files returned as task output and for files returned from the API.  Files from the API will have a `bucket` and `filepath`, while a file returned as the output of a task will have a `filename`.

* Verifying task output here - https://github.com/nasa/cumulus/blob/7b4793988a98d68e7fbad9b0d699798ae4c3686c/example/spec/parallel/ingestGranule/IngestGranuleDuplicateHandlingSpec.js#L113
* Verifying response from the API here - https://github.com/nasa/cumulus/blob/5e868fa749ffe5b1c8d67ea0261f0358d1d3825c/example/spec/parallel/ingestGranule/IngestGranuleSuccessSpec.js#L464